backend/api.test/EndpointTest.cs

```diff
@@ -190,7 +190,8 @@ public async Task StartMissionTest()
             var query = new ScheduledMissionQuery
             {
                 RobotId = robotId,
-                AssetCode = "test",
+                AssetCode = "JSV",
```

Comment:
```suggestion
                AssetCode = "test",
```

Unless the `AssetCode` is required to be `JSV` I would put it as an anonymous name.

Reply:
Or even better, use the arrange setup which is used in the "AreaTest" function.